Extensions/SystemInfo/systeminfotools.js

```diff
@@ -28,3 +28,17 @@ gdjs.evtTools.systemInfo.isMobile = function() {
 
 	return false;
 };
+
+/**
+ * The device supports WebGL
+ * @param {gdjs.RuntimeScene} runtimeScene
+ * @returns {Boolean} WebGL is supported
```

Comment:
Boolean (with a capital B) is said to be a "boxed primitive". Basically, never use it - always use the type "boolean" in JSdoc (more information here: http://adripofjavascript.com/blog/drips/the-difference-between-boolean-objects-and-boolean-primitives-in-javascript.html).

```suggestion
 * @returns {boolean} true if WebGL is supported
```